src/cloud-code/Parse.Cloud.js

```diff
@@ -528,6 +529,34 @@ ParseCloud.beforeConnect = function (handler, validationHandler) {
   );
 };
 
+/**
+ * Sends an email through the Parse Server mail adapter.
+ *
+ * **Available in Cloud Code only.**
+ * **Requires a mail adapter to be configured for Parse Server.**
+ *
+ * ```
+ * Parse.Cloud.sendMail({
+ *   from: 'Example <test@example.com>',
+ *   to: 'contact@example.com',
+ *   subject: 'Test email.',
+ *   text: 'This email is a test'
+ * });
+ *```
+ *
+ * @method sendMail
+ * @name Parse.Cloud.sendMail
+ * @param {Object} data The object of the mail data to send
+ */
+ParseCloud.sendMail = function (data) {
+  const config = Config.get(Parse.applicationId) || {};
+  const emailAdapter = config.userController.adapter;
+  if (!emailAdapter) {
+    throw 'Failed to send email because no mail adapter is configured for Parse Server.';
```

Comment:
Is it fine to throw here? Where is that caught?

Reply:
Won't get caught unless Parse.Cloud.sendMail is wrapped in a try / catch. How else do you think it should it be handled? `return` and add a log?